### DISCUSSION TITLE
Very high data usage?

### DISCUSSION THREAD

**Discussion by FlaMtnBkr** [OrcaSlicer/OrcaSlicer#7653] [General]
I updated an "unsupported" i7-7700 computer to Windows 11 a few weeks ago and have been trying to figure out some issues with my Wi-Fi adapter(s) and while poking around settings I found data usage broken down to individual programs. I was shocked to see Orca Slicer in the #1 spot with just under 3 gb of data in the last 30 days?! And a second adapter has just under 2.5 gb. I haven't been printing all that much recently and all the prints have been fairly simple brackets and similar designs I made and no complex organic shaped prints. 

This seems fairly extreme?! Could there be that much data going to the printer? Or is there any other reason there would be that much? Could it be false reporting somehow?

The adapter that came with the computer used to get poor reception so I got a USB adapter to get better speeds but have since installed a mesh system so I guess the original adapter has a good signal again and part of what I was trying to figure out. I just started looking at things and I'm not even sure that both adapters are communicating with the internet and one might only be connecting to the LAN? 

Does any of this sound normal or should I be concerned? I'm using portable v2.20 that I downloaded from GitHub.

**Comment by FlaMtnBkr** [comment]
No thoughts?

Most g-code files sent to the ~~computer~~ **_printer_** are only a couple Mb in size and I'm not sure what else Orca Slicer connects to the internet for other than connecting to Bambu servers? So 5 Gb of data in 30 days seems pretty extreme and I've found no reason to not believe W11 network data usage numbers.

Has anyone else checked to see how much their computers are reporting?

I would think when a program has unusually high data  usage, it would be a concern for anyone using the same program, but maybe not...

**Reply by flyfr8rs** [reply]
I have just encountered an even more extreme case of high data consumption by OrcaSlicer. W11 reports 67GB (yes, GB) of internet data consumed by OrcaSlicer in the past 7 days....WTF? I've only printed a few (<10) small (<100KB) .3MF files in that time frame (on a BambuLab X1C). I recently updated to v2.2.0, but have been on Windows 11 for over a year, and noticed no prior excessive data usage by OrcaSlicer. Something is terribly wrong - I'd appreciate any guidance on how to resolve this.

I may have left the video feed open for a few days inadvertently - is it possible to have consumed that much data from the video stream??

**Comment by pgifford** [comment]
Huh, same...interesting.  I'm on Windows 10 running the nightly build (v2.3.0-dev) which I update if something looks applicable to my setup (e.g. I'm not going to update just to get the latest Russian translation 😆).  Maybe it's the video feed?  Sometimes I keep that open on my desktop.  I have 3 printers going much of the time though I rarely have more than 1 instance of OrcaSlicer open and I never stream the feed from more than 1 printer.

<img width="351" alt="network" src="https://github.com/user-attachments/assets/0624842a-7c30-4c13-845d-07c665efec51" />

The high usage for 7-Zip and explorer are probably because I keep a lot of models on my NAS and a few of the creators I support post zip files and I will unzip archives located on the NAS.  Anything I print is first moved to a local disk so OrcaSlicer never talks to the NAS.

Welp, I've always said OrcaSlicer is the number 1 slicer and now I have numbers to prove it!  🤣

**Comment by FlaMtnBkr** [comment]
It would be nice to know if things like this are ever explored or if they mainly get overlooked unless a bug report?

I know there aren't many devs but even just a thumbs up or a looking into it would be helpful...

I didn't see a reply for a while so didn't check back. At least I wasn't the only one.

I don't usually leave the video feed going. Occasionally I will check it real quick to make sure the print is coming along and not messed up but I usually stop it, though I certainly could have left it running inadvertently. Even a low res video feed doesn't seem like it would add up to some of these numbers? I also have my computer set to go to sleep after an hour though sometimes it doesn't happen and I have no idea why. But it usually seems to go to sleep which seems like it would halt any data transfer.

I'm not sure which version this happened with but based on the date I'd assume 2.2.0 which I recently found a bug that effects the scarf seam. I need to submit it but a lot is required to do so and I haven't yet. I've been using 2.1.1 but I also haven't been checking the data usage very often.

Anyways, at least I wasn't crazy...

**Comment by Plumpkatt1** [comment]
I'm experiencing the same thing and it's usually during slicing of the file. I'm slicing a large file currently and I'm on Windows 10 and I'm noticing high GPU usage which is normal high CPU usage which is normal but I'm also experiencing my Wi-Fi reporting that I'm downloading anywhere from 8 to 12 megabits a second throughout this process which doesn't make sense to me.

**Comment by sgofferj** [comment]
How about you all Wireshark the Windows interface and see what kind of traffic it is and where it goes?

Edit:
If you're on Windows - are you sure you got the genuine Orcaslicer and not some version with a Trojan or something?
I'm running Linux and I quickly checked the stats of my network switch and the IDS. I don't see anything out of the ordinary on the port of my office PC...
